verify_rpms/rpm_verifier.py

```diff
@@ -147,8 +146,8 @@ def __call__(self, img: str) -> ProcessedImage:
     required=True,
 )
 @click.option(
-    "--fail-unsigned",
-    help="Exit with failure if unsigned RPMs found",
+    "--fail-on-errors",
+    help="Exit with failure if unsigned RPMs or other errors were found",
```

Comment:
Let's not change the input params at this point. You can change the help, but keep the option name unchanged.

Reply:
ack, fixed (although I find it a bit hard to understand)

Reply:
There's already a task on another repo that uses those option names.

Reply:
`fail-on-errors` or `fail-unsigned`? 
I changed it back to `fail-unsigned`, which was there originally

Reply:
The original. I was mentioning this as the reason not to change the way we run the script.